earthfile2llb/converter.go

```diff
@@ -39,11 +42,8 @@ import (
 	"github.com/earthly/earthly/util/syncutil/semutil"
 	"github.com/earthly/earthly/variables"
 	"github.com/earthly/earthly/variables/reserved"
-
```

Comment:
we try to keep all earthly imports separate from third party libraries -- can you restore this newline.

Reply:
@r-LaForge did you back this change out? I'm still seeing it removed in the latest PR.

Reply:
I think I have the ordering we want now.